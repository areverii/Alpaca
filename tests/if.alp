function void main(<string> args):
    bool x
    if true:
        prints("Should print")
    end
    if false:
        prints("Should not print")
    end
    if x:
        prints("Should not print")
    end
    else:
        prints("Should print")
    end
    if not x:
        prints("Should print")
    end
end
