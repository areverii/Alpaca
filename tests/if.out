Should print
Should print
Should print
